Main.py

```diff
@@ -1,19 +1,29 @@
 import utils
 import time
 import math
+import threading
+from threading import Thread
 
+# CONSTANTS # # # # # # # #
+SELL_THRESHOLD = -0.08    # % drop/gain in the timespan of CHECK_TIMER
+CHECK_TIMER = 10          # calculates the drop/gain over the last X seconds
+# # # # # # # # # # # # # #
+
+state = {}
```

Comment:
This variable will keep track of which exchange we made purchases on and which has yet to be sold

Reply:
we can also store some other sweet state variables in here if it makes things easier